bust-a-move-framework/src/main/java/bustamove/App.java

```diff
@@ -93,15 +93,6 @@ public static final Game getGame() {
     }
 
     /**
-     * Setter function for game object.
-     *
-     * @param g game object
-     */
```

Comment:
Wasn't used and was listed as a bug by FindBugs. We aren't supposed to write to a static field from an instance method. We also aren't supposed to instantiate the game object in there, but I left that since we are going to apply the singleton pattern on the game class anyways.